rosidl_generator_cpp/rosidl_generator_cpp/__init__.py

```diff
@@ -190,6 +190,10 @@ def primitive_value_to_cpp(type_, value):
         return '%sul' % value
 
     if type_.typename == 'int64':
+        # Handle edge case for INT64_MIN
+        # See https://en.cppreference.com/w/cpp/language/integer_literal
+        if -9223372036854775808 == value:
+            return '(%s - 1)' % (value + 1)
```

Comment:
Shouldn't the `%s` be followed by a `ll`?

Reply:
Yes, e1c1b28